src/app/globals.css

```
@@ -2,26 +2,16 @@
 @tailwind components;
 @tailwind utilities;
 
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
-}
-
-@media (prefers-color-scheme: dark) {
-  :root {
-    --foreground-rgb: 255, 255, 255;
-    --background-start-rgb: 0, 0, 0;
-    --background-end-rgb: 0, 0, 0;
-  }
-}
-
-body {
-  color: rgb(var(--foreground-rgb));
-}
-
 @layer utilities {
   .text-balance {
     text-wrap: balance;
   }
 }
+
+@layer base {
+  html {
+    font-family: "AppleSDGothicNeo", "Noto Sans KR", sans-serif;
+    font-size: 14px;
```

우리 컨벤션 정할때 px대신 rem으로 하기로 했던거 기억나??

px은 절대단위고, rem은 최상위 요소의 글꼴 크기에 상대적으로 크기가 변하는 상대단위야
font-size를 다 절대 단위로 박아버리면 크기가 고정되어 모든 브라우저 설정을 덮어버려서 font-size을 키우고 싶어도 못 키우고,,줄이고 싶어도 못 줄여. 나도 이걸 안지는 얼마 되지 않았는데 안 이후로는 최대한 rem을 사용해서 사용자 경험 개선에 도움을 주려하는 편이야. 최상단에서 62.5% 를 주면 px단위에서 1/10배의 값이 rem의 값이어서 편하게 적용할 수 있어~!

```suggestion
    font-size: 62.5%;
```